hbase-server/src/main/java/org/apache/hadoop/hbase/util/LossyCounting.java

```diff
@@ -119,16 +138,33 @@ public long getDataSize() {
     return data.size();
   }
 
-  public boolean contains(String key) {
+  public boolean contains(T key) {
     return data.containsKey(key);
   }
 
-  public Set<String> getElements(){
+  public Set<T> getElements(){
     return data.keySet();
   }
 
   public long getCurrentTerm() {
     return currentTerm;
   }
+
+  class SweepRunnable implements Runnable {
+    @Override public void run() {
+      if (LOG.isTraceEnabled()) {
+        LOG.trace("Starting sweep of lossyCounting-" + name);
+      }
+      try {
+        sweep();
+      } catch (Exception exception) {
+        LOG.debug("Error while sweeping of lossyCounting-{}", name, exception);
```

Comment:
(damnit, lost the previous comment)

Make sure you're getting the right `debug` method here. The `debug(String, Object, Exception)` not `debug(String, Object, Object)`. Hit this the other day.

Reply:
Fixing